platforms/core-runtime/logging/src/main/java/org/gradle/internal/logging/text/TreeFormatter.java

```diff
@@ -263,7 +263,7 @@ private void writeNode(Node node) {
         }
 
         StyledTextOutput output = new LinePrefixingStyledTextOutput(original, node.prefix, false);
-        if (!node.valueWritten) {
+        if (!node.valueWritten && !"".contentEquals(node.value)) {
```

Comment:
What's the effect of this?

Reply:
This came about from tweaking the output format in the dependency insight report when I was experimenting with the opt-in flag for the WoT.  It may not be necessary now, but it might be what removed some empty lines from the output in `customLogger.out`.  

I'll try removing this and see what happens.

Reply:
Seems to work.